src/wallet/test/wallet_test_fixture.h

```diff
@@ -19,10 +19,13 @@ namespace wallet {
 /** Testing setup and teardown for wallet.
  */
 struct WalletTestingSetup : public TestingSetup {
-    explicit WalletTestingSetup(const std::string& chainName = CBaseChainParams::MAIN);
+    WalletTestingSetup() : WalletTestingSetup(CBaseChainParams::MAIN) { }
+    explicit WalletTestingSetup(const std::string& chainName);
+    WalletTestingSetup(const WalletTestingSetup&) = delete;
+    WalletTestingSetup(WalletTestingSetup&&) = delete;
     ~WalletTestingSetup();
 
-    std::unique_ptr<interfaces::WalletLoader> m_wallet_loader = interfaces::MakeWalletLoader(*m_node.chain, *Assert(m_node.args));
+    std::unique_ptr<interfaces::WalletLoader> m_wallet_loader;
```

Comment:
I presume this is to silence clang TS warnings? If so, could mention it in the commit body.

Reply:
`Assert` uses `__func__` which is not defined outside of a function body, but the initializer list counts as part of the constructor body.